copt/constraint.py

```diff
@@ -86,7 +77,19 @@ def lmo(self, u, x):
             u[largest_coordinate]
         )
 
-        return update_direction, 1
+        return update_direction, int(largest_coordinate), None, 1
+
+    def is_vertex(self, x):
```

Comment:
I don't see where the is_vertex functions are being used in the code (but I could be missing something)

Reply:
Removed